test/parallel/test-cli-node-options-docs.js

```diff
@@ -25,6 +25,8 @@ const v8OptionsText = cliText.match(/<!-- v8-options start -->(.*)<!-- v8-option
 const manPage = path.join(rootDir, 'doc', 'node.1');
 const manPageText = fs.readFileSync(manPage, { encoding: 'utf8' });
 
+const ignoredDocs = ['--experimental-quic'];
```

Comment:
I'm not sure we need another way of marking an option as undocumented. We have other undocumented command line options which don't need this additional code. 

e.g. 
- https://github.com/nodejs/node/blob/b7ce941f9ab9b301c128f5c6e4ec357fa56023e5/src/node_options.cc#L820-L823
- https://github.com/nodejs/node/blob/b7ce941f9ab9b301c128f5c6e4ec357fa56023e5/src/node_options.cc#L1037-L1040

Reply:
Makes sense. I'll update it.